core/src/main/java/feast/core/service/JobService.java

```diff
@@ -99,19 +93,23 @@ public ListIngestionJobsResponse listJobs(ListIngestionJobsRequest request) {
         // multiple filters can apply together in an 'and' operation
         if (!filter.getStoreName().isEmpty()) {
           // find jobs by name
-          List<Job> jobs = this.jobRepository.findByJobStoresIdStoreName(filter.getStoreName());
+          List<Job> jobs = this.jobRepository.findByJobStoreName(filter.getStoreName());
           Set<String> jobIds = jobs.stream().map(Job::getId).collect(Collectors.toSet());
           matchingJobIds = this.mergeResults(matchingJobIds, jobIds);
         }
         if (filter.hasFeatureSetReference()) {
           // find a matching featuresets for reference
-          FeatureSetReference fsReference = filter.getFeatureSetReference();
+          FeatureSetReferenceProto.FeatureSetReference fsReference =
+              filter.getFeatureSetReference();
 
           // find jobs for the matching featuresets
           Collection<Job> matchingJobs =
-              this.jobRepository
-                  .findByFeatureSetJobStatusesFeatureSetNameAndFeatureSetJobStatusesFeatureSetProjectName(
-                      fsReference.getName(), fsReference.getProject());
+              this.jobRepository.findByFeatureSetReference(
+                  FeatureSetReference.of(
+                      fsReference.getProject().isEmpty()
+                          ? Project.DEFAULT_NAME
```

Comment:
Should this be moved into the reference? So that the caller doesnt have to implement the handling of default project name.